<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>689</width>
    <height>148</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Syncthing .stversions restore utility</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" rowspan="4">
     <widget class="QLabel" name="label_logo">
      <property name="minimumSize">
       <size>
        <width>130</width>
        <height>130</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>130</width>
        <height>130</height>
       </size>
      </property>
      <property name="pixmap">
       <pixmap resource="logo.qrc">:/logo/syncthing.png</pixmap>
      </property>
     </widget>
    </item>
    <item row="0" column="1" rowspan="4">
     <spacer name="horizontalSpacer_logo">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>7</width>
        <height>118</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="2" colspan="7">
     <widget class="QLineEdit" name="lineEdit_stversions">
      <property name="placeholderText">
       <string>.stversions folder address :</string>
      </property>
     </widget>
    </item>
    <item row="0" column="9">
     <widget class="QPushButton" name="pushButton_Browse">
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="label_from">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Restore from :</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3" colspan="2">
     <widget class="QDateTimeEdit" name="dateTimeEdit_from">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="displayFormat">
       <string>yyyy/MM/dd HH:mm:ss</string>
      </property>
     </widget>
    </item>
    <item row="1" column="5" colspan="2">
     <spacer name="horizontalSpacer_from_to">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>168</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="7">
     <widget class="QLabel" name="label_to">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>To :</string>
      </property>
     </widget>
    </item>
    <item row="1" column="8" colspan="2">
     <widget class="QDateTimeEdit" name="dateTimeEdit_to">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="displayFormat">
       <string>yyyy/MM/dd HH:mm:ss</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2" colspan="2">
     <widget class="QCheckBox" name="checkBox_restoreAll">
      <property name="text">
       <string>Restore everything</string>
      </property>
     </widget>
    </item>
    <item row="2" column="4" colspan="2">
     <spacer name="horizontalSpacer_checks">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>74</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="6" colspan="4">
     <widget class="QCheckBox" name="checkBox_overwrite">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Overwrite if the destination is already taken</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2" colspan="8">
     <widget class="QPushButton" name="pushButton_Start">
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="logo.qrc"/>
 </resources>
 <connections/>
</ui>
